src/test/ui/proc-macro/amputate-span.rs

```diff
@@ -0,0 +1,63 @@
+// aux-build:amputate-span.rs
```

Comment:
Should we try and make this one `run-rustfix`?

Reply:
done in follow up commit, thank you for pointing that out!